bundles/org.openhab.binding.buienradar/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```

Comment:
Can you please remove the .settings directory? We have put that on the git ignore list recently as well.

Reply:
Done